deluge/plugins/Notifications/deluge/plugins/notifications/gtkui.py

```diff
@@ -159,15 +158,14 @@ def __blink(self):
         return defer.succeed(_("Notification Blink shown"))
 
     def __popup(self, title='', message=''):
-        import gtk
         if not self.config['popup_enabled']:
             return defer.succeed(_("Popup notification is not enabled."))
         if not POPUP_AVAILABLE:
             return defer.fail(_("pynotify is not installed"))
 
-        if pynotify.init("Deluge"):
-            icon = gtk.gdk.pixbuf_new_from_file_at_size(deluge.common.get_pixmap("deluge.svg"), 48, 48)
-            self.note = pynotify.Notification(title, message)
+        if Notify.init("Deluge"):
+            icon = GdkPixbuf.Pixbuf.new_from_file_at_size(deluge.common.get_pixmap("deluge.svg"), 48, 48)
+            self.note = Notify.Notification(title, message)
```

Comment:
This might need extra code, looking at similar changes in another project: https://github.com/eonpatapon/mpDris2/commit/c1a7e9741f419836b307d8d1514a68350da620ea#diff-04de6ed057889d6b920e21aa4aa92919R790